### PR TITLE
Fix editing created at of article

### PR DESCRIPTION
**Pull request by RostislavKreisinger**
| Q             | A
| ------------- | ---
|Description, reason for the PR| created at of article is editable
|New feature| No 
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| No 
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: project-base/src/Model/Article/ArticleDataFactory.php [hunk 1]
```diff
@@ -20,4 +21,15 @@ protected function createInstance(): BaseArticleData
...
+    /**
+     * @param \App\Model\Article\ArticleData $articleData
+     * @param \App\Model\Article\Article $article
+     */
+    protected function fillFromArticle(BaseArticleData $articleData, Article $article): void
+    {
+        parent::fillFromArticle($articleData, $article);
+
+        $articleData->createdAt = $article->getCreatedAt();
+    }
```
Comment (TomasLudvik) [inline project-base/src/Model/Article/ArticleDataFactory.php:34]: You should fix this in framework class

**Review by TomasLudvik — CHANGES_REQUESTED** [review]

File: project-base/src/Model/Article/ArticleDataFactory.php (hunk 1 above)
Comment (RostislavKreisinger) [reply project-base/src/Model/Article/ArticleDataFactory.php:34]: No, here is right place. Attribute createdAt in article would be added in the base project as an extension example, so having half the implementation in the framework and half in projet-bese is not possible.

File: project-base/config/bundles.php [hunk 2]
```diff
@@ -24,7 +24,6 @@
     Stof\DoctrineExtensionsBundle\StofDoctrineExtensionsBundle::class => ['all' => true],
     Snc\RedisBundle\SncRedisBundle::class => ['all' => true],
     Symfony\Bundle\FrameworkBundle\FrameworkBundle::class => ['all' => true],
-    # JMS TranslationBundle needs to be after Symfony Framework as Symfony Framework also adds translate commands
```
Comment (TomasLudvik) [inline project-base/config/bundles.php:27]: Please return this comment, it is important

File: upgrade/UPGRADE-v11.0.0-dev.md [hunk 3]
```diff
@@ -875,6 +875,8 @@ There you can find links to upgrade notes for other versions too.
     - replace usages of `SitemapRepository::getSitemapItemsForVisibleProducts()` with `SitemapRepository::getSitemapItemsForListableProducts()` in your project as old method no longer exists
 - session is valid for a year and the cart is now only deleted after 130 days from the user's last activity ([#2537](https://github.com/shopsys/shopsys/pull/2537))
     - see #project-base-diff to update your project
+- Fix editing attribute created at of article, attribute created at move from project base to framework ([#2546](https://github.com/shopsys/shopsys/pull/2546))
```
Comment (TomasLudvik) [inline upgrade/UPGRADE-v11.0.0-dev.md:878]:
```suggestion
- Fix editing attribute created at of article, attribute created at has been moved from project base to framework ([#2546](https://github.com/shopsys/shopsys/pull/2546))
```

**Review by TomasLudvik — APPROVED** [review]

**Review by pk16011990 — APPROVED** [review]

**Pull request merged**